.packit.yaml

```diff
@@ -112,18 +120,17 @@ jobs:
               BusinessUnit: sst_upgrades@leapp_upstream_test
   env:
```

Comment:
the idea was to put it there to make copying easier / creating a new one easier for people. But honestly I've been not sure whether it's helpful or not. So I am pretty ok to drop it

Reply:
It gives false comfort that you don't need to define the fields that are already defined in abstract templates - and this may cause future mistakes and complicate review process. So I am in favor of removing it.

Reply:
done